packages/react-core/src/demos/examples/AlertGroup/AlertGroupToastWithNotificationDrawer.tsx

```diff
@@ -316,15 +315,15 @@ export const AlertGroupToastWithNotificationDrawer: React.FunctionComponent = ()
       isNotificationDrawerExpanded={isDrawerExpanded}
     >
       <PageSection>
-        <TextContent>
-          <Text component="h1">Alert Group with Notification Drawer demo</Text>
-          <Text component="p">
+        <Content>
+          <h1>Alert Group with Notification Drawer demo</h1>
+          <p>
             New alerts can be added with buttons below. Each alert has a timeout of 7 seconds, however, even after the
             timeout expires, all alerts are still visible in the notification drawer. By default, only 3 alerts are
             displayed. The rest can be accessed in the notification drawer after clicking on the bell icon in the header
             or by clicking on the overflow message.
```

Comment:
```suggestion
            New alerts can be added with the following buttons. Each alert has a timeout of 7 seconds, however, even after the timeout expires, all alerts are still visible in the notification drawer. By default, only 3 alerts are displayed. The rest can be accessed in the notification drawer after clicking on the bell icon in the header or by clicking on the overflow message.
```
just small nitpick edits that I notice haha